pkg/plugins/platform/router.go

```diff
@@ -0,0 +1,92 @@
+package platform
```

Comment:
This could probably go outside of the platform package, as it's not plugin-specific and it doesn't look like it needs to access app data. I'm not advocating another package be created for it, but perhaps we should start consolidating into the ddevapp package.